src/components/CreatePost/StepperButton.tsx

```diff
@@ -2,43 +2,53 @@ import { ArrowBackIosNew, ArrowForwardIosOutlined } from '@mui/icons-material';
 import { Button, Typography } from '@mui/material';
 import { useRouter } from 'next/router';
 import { Dispatch, SetStateAction } from 'react';
+import { UseFormHandleSubmit } from 'react-hook-form';
+
+import { createPost } from '@/api/createPost';
+import { CreatePost } from '@/types/CreatePost';
 
 interface StepperButtonProps {
   format: 'forward' | 'backward' | 'complete';
   steps: number;
   setSteps: Dispatch<SetStateAction<number>>;
+  onClick?: UseFormHandleSubmit<CreatePost>;
 }
 
-const StepperButton = ({ format, steps, setSteps }: StepperButtonProps) => {
+const StepperButton = ({ format, steps, setSteps, onClick }: StepperButtonProps) => {
   const router = useRouter();
 
+  const onSubmit = (data: CreatePost) => {
+    console.log(data);
+    createPost({ ...data, thumbnail: '123' }, '178');
+  };
+
   return (
     <>
       {format === 'forward' ? (
         <Button
           variant='contained'
-          sx={{ display: 'flex', alignItems: 'center' }}
-          onClick={() => setSteps(steps + 1)}>
+          sx={centered}
+          type='submit'
+          onClick={() => {
+            if (onClick) {
+              onClick(onSubmit)();
+              setSteps(steps + 1);
+            }
+          }}>
           <Typography variant='body1' component='span' sx={{ fontSize: '0.5rem' }}>
             다음
           </Typography>
           {<ArrowForwardIosOutlined sx={{ fontSize: '0.5rem' }} />}
         </Button>
       ) : format === 'backward' ? (
-        <Button
-          variant='contained'
-          sx={{ display: 'flex', alignItems: 'center' }}
-          onClick={() => setSteps(steps - 1)}>
+        <Button variant='contained' sx={centered} onClick={() => setSteps(steps - 1)}>
           {<ArrowBackIosNew sx={{ fontSize: '0.5rem' }} />}
```

Comment:
ask;

```suggestion
          <ArrowBackIosNew sx={{ fontSize: '0.5rem' }} />
```

제거해도 되는 중괄호 아닌가요? 혹시 사용하는 이유가 있나요? (41번 로직도 같아요)

Reply:
저부분이 format에 따라서 변경 가능하게 하려고 했었는데
지금은 필요 없네요!